v2/captions.go

```diff
@@ -1,40 +1,17 @@
-package threeplay
+package v2
 
 import (
 	"errors"
 	"fmt"
+	"github.com/NYTimes/threeplay/common"
```

Comment:
the latest version of the package is now using ``nytimes`` all lower case. This should fix the build.
```suggestion
	"github.com/nytimes/threeplay/common"
```